src/envo/envo-edit.owl

```diff
@@ -3535,6 +3553,15 @@ Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510038>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510039>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510040>))
 Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510041>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510055>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510056>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510057>))
+Declaration(Class(<http://purl.obolibrary.org/obo/ENVO_03510058>))
```

Comment:
Consider the comments above - same issues here. There should be an ink hierarchy, probably dovetailing with the dye hierarchy, focusing on the actual chemical properties of these materials (which include aromatic groups or similar that absorb visible light)

Reply:
Perhaps improving the definition of "ink" ENVO:03510065 and moving this under it will help.